.claude/hooks/claim-verifier.sh

```
@@ -67,14 +67,18 @@ fi
 # Extract PR body — gh pr create uses --body flag or stdin
 # ---------------------------------------------------------------------------
 
-# Try --body flag first
-BODY=$(echo "$CMD" | grep -oE '\-\-body[[:space:]]+'"'"'[^'"'"']*'"'"'([^[:space:]]+)?' | \
-       sed "s/--body[[:space:]]*'"'"'//; s/'"'"'$//" | tr '\n' ' ' | sed 's/[[:space:]]*$//')
+# Try --body flag first (single-quoted body)
+MATCH=$(echo "$CMD" | grep -oE '\-\-body[[:space:]]+'"'"'[^'"'"']*'"'"'([^[:space:]]+)?' 2>/dev/null || true)
+if [ -n "$MATCH" ]; then
+  BODY=$(echo "$MATCH" | sed "s/--body[[:space:]]*'"'"'//; s/'"'"'$//" | tr '\n' ' ' | sed 's/[[:space:]]*$//')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)